src/styles/_grid.sass

```diff
@@ -141,11 +141,6 @@
           left: $perc
         
         $i: $i + 1
-      
-    
-  
-
 
 .row.full-height
-  min-height: 100vh
-
+  min-height: 100vh
```

Comment:
It's super hard for me to tell because I visually see the layout, but would it be beneficial to leave `min-height: 100%` as a fallback if a browser doesn't support viewport units?  Granted, those browsers should just die already, but in that case it might at least keep the row at least as high as its container if the container has other contents affecting its height.

The class `full-height` is also a little confusing in this context.  Full-height of its parent container, or the viewport?  Maybe have `.row.full-height` be `min-height: 100%` and `.row.viewport-height` be min-height: 100vh`?  Dunno if that's too redundant since they just represent specific CSS properties.